src/EnergyPlus/DElightManagerF.cc

```diff
@@ -803,6 +799,9 @@ namespace DElightManagerF {
         NewAspectRatio = 1.0;
 
         if (state.dataInputProcessing->inputProcessor->getNumObjectsFound(state, CurrentModuleObject) == 1) {
+            int NAlphas;
+            int NNum;
+            int IOStat;
```

Comment:
[src/EnergyPlus/DElightManagerF.cc:795]:(style),[variableScope],The scope of the variable 'NAlphas' can be reduced.
[src/EnergyPlus/DElightManagerF.cc:796]:(style),[variableScope],The scope of the variable 'NNum' can be reduced.
[src/EnergyPlus/DElightManagerF.cc:797]:(style),[variableScope],The scope of the variable 'IOStat' can be reduced.